cmd/sovereignnode/chainSimulator/sovereignChainSimulator_test.go

```diff
@@ -4,13 +4,13 @@ import (
 	"testing"
 	"time"
 
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/stretchr/testify/require"
```

Comment:
goimport.

Reply:
This seems to be ok, it's split in 3 categories.